bot/code_review_bot/config.py

```diff
@@ -49,6 +49,7 @@ def __init__(self):
         self.taskcluster = None
         self.try_task_id = None
         self.try_group_id = None
+        self.hgmo_cache = tempfile.mkdtemp(suffix="hgmo")
```

Comment:
We are not going to cleanup after ourselves, but this should be fine in our current environment.

Reply:
I added a cleanup `atexit`